tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+        "es5",
+        "es6",
+        "es2015.collection",
+        "es2015.iterable",
+        "es2015.core",
+        "dom"
+      ],
+      "types": [
+        "node"
+      ],
+      "target": "es2015",
+      "strict": true
+    },
+    "include": [
+      "examples/*.ts",
+    ]
+  }
```

Comment:
Style nit: it looks like this is indented four spaces at the outer level, but two spaces internally

Reply:
Done